packages/@aws-cdk/aws-ec2/lib/bastion-host.ts

```diff
@@ -90,8 +90,9 @@ export interface BastionHostLinuxProps {
  * You can also configure this bastion host to allow connections via SSH
  *
  * @experimental
+ * @resource AWS::EC2::Instance
  */
-export class BastionHostLinux extends Construct implements IInstance {
+export class BastionHostLinux extends Resource implements IInstance {
```

Comment:
Should `IInstance` extend `IResource`?

Reply:
[It already does](https://github.com/aws/aws-cdk/blob/84a3ef65fcc475fb5d59522ffd0d46cf77c78e8a/packages/%40aws-cdk/aws-ec2/lib/instance.ts#L20-L20), hence why this change was needed here.